### DISCUSSION TITLE
[blocks]: finance dashboard

### DISCUSSION THREAD

**Discussion by MaestroDev19** [shadcn-ui/ui#3158] [Blocks Request]
### Description

A dashboard for users to view their budgets, finance course(video or article), transactions and others

**Comment by birobirobiro** [comment]
https://www.badget.io/

Preview: https://twitter.com/CodeHagen/status/1767648655123546621

**Comment by itsjoeoui** [comment]
I'm building https://fijoy.app (open-source)! It is still quite early stage but it will get there!
Will be looking for more feedback once I build it out more :) Feel free to email me at joey@fijoy.app too
https://github.com/itsjoeoui/fijoy 😄
